### PR TITLE
Base-Spells: Add trader magic

### PR DESCRIPTION
**Pull request by jandersson**

### CODE REVIEW

**Comment by jandersson** [comment]
Adding spells as I go. Will leave a message when done

**Comment by jandersson** [comment]
All spells (except metamagic/metaspells) added

**Comment by jandersson** [comment]
Going through the spells now and updating cast messaging

File: data/base-spells.yaml [hunk 1]
```diff
@@ -1559,6 +1647,11 @@ spell_data:
     abbrev: TR
     mana: 5
     cyclic: true
+  Turmar Illumination:
+    abbrev: TU IL
```
Comment (rpherbig) [inline data/base-spells.yaml]: https://elanthipedia.play.net/Turmar_Illumination says the abbrev here is `TURI`. Could you double check this in game?

Comment (bmcneff) [reply data/base-spells.yaml]: Validated that the abbrev is TU IL

Comment (bmcneff) [reply data/base-spells.yaml]: Technically both work:

>prep TURI 5
You raise your hands skyward, chanting the equation of the Turmar Illumination spell.

Comment (jandersson) [reply data/base-spells.yaml]: Thanks for that. We should probably use the official abbreviation

**Pull request merged**